### PR TITLE
Add statistics section

### PR DESCRIPTION
**Pull request by jianoway**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -26,6 +28,36 @@ const Home = (props) => {
     });
   }, []);
 
+  const fetchMembers = async () => {
+    try {
+      // assume API call returns array of members, but for now i'll use the arr of events
+      const response = await request.get("events/")
```
Comment (jerrenss) [inline pages/index.js:34]: For now, let's not use API calls as we are going to temporarily decouple the backend. All API calls have been removed in the latest master. It's fine to use constant values for the stats.

File: pages/index.js [hunk 2]
```diff
@@ -229,6 +261,33 @@ const Home = (props) => {
 
         <div className={styles.contactUs} id="contact">
           <FadeInSection>
+          <script>
```
Comment (jerrenss) [inline pages/index.js:264]: Shall we have the statistics displayed near the top of the screen, below About Us? Feel free to use these for inspiration:

image

image

**Review by jerrenss — CHANGES_REQUESTED** [review]
Other than that all's good, nice work!!

**Comment by jianoway** [comment]
I added the statistics section to below the About Us section. The background image used is from [here](https://unsplash.com/photos/17_tB-oI0ao) and is licensed to be used for commercial and non-commercial purposes for free. The image is just placeholder and can be changed to a more appropriate one. Please let me know how this iteration looks, thanks!

**Comment by jerrenss** [comment]
It looks much better! Since we are going to deploy this, let's not use the placeholder image as it dosen't fit the overall look. Let's go for a more minimalist design first? Something like this.

image

Changes:
- Remove fixed width
- Add border radius of 8px
- Change fonts to black
- Use background-color #F5F5F5 instead of background-image
- Remove <h1/> header

As an optional added challenge, try to make this mobile responsive. Such that when the user views it on his phone, it still looks decent. Can use Chrome Dev Tools or manually shrink the browser to test this

**Comment by jianoway** [comment]
Thanks for the CountUp suggestion, wasn't aware of the existence of the library until now!

**Comment by jerrenss** [comment]
Great work!! Am trying to get the actual stats from the exco. Will approve and merge once the numbers are confirmed :)

**Review by jerrenss — APPROVED** [review]

**Pull request merged**